dcc-portal-ui/app/scripts/downloader/js/request.js

```diff
@@ -136,8 +136,10 @@
 
         $scope.params.processing = false;
       },function (error) {
-        $scope.params.processing = false;
-        $scope.params.downloadEnabled = false;
+        if(error.status === 503){
+          $scope.params.processing = false;
+          $scope.params.downloadEnabled = false;
+        }
```

Comment:
I think this maybe the only way:

https://github.com/mgonto/restangular/issues/566#issuecomment-34312971

see 

https://github.com/mgonto/restangular/issues/566#issuecomment-34453140